### PR TITLE
WIP - Update bazel prefix for 3.4.1

### PR DESCRIPTION
**Pull request by liggitt**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

The current value used for the importmap doesn't work with bazel 3.4.1 (updated in https://github.com/kubernetes/kubernetes/pull/96011).

running push-build with bazel 2.2.x:
```
package conflict error: k8s.io/kubernetes/vendor/k8s.io/component-base/metrics: multiple copies of package passed to linker:
	//staging/src/k8s.io/component-base/metrics:go_default_library
	//vendor/k8s.io/component-base/metrics:go_default_library
Set "importmap" to different paths or use 'bazel cquery' to ensure only one
package with this path is linked.
This will be an error in the future.
```

with 3.4.1:
```
link: package conflict error: k8s.io/kubernetes/vendor/k8s.io/component-base/metrics: multiple copies of package passed to linker:
	//staging/src/k8s.io/component-base/metrics:go_default_library
	//vendor/k8s.io/component-base/metrics:go_default_library
```

Requires https://github.com/kubernetes/kube-openapi/pull/222 first

/cc @mikedanese 

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -510,7 +510,7 @@ replace (
 	k8s.io/klog/v2 => k8s.io/klog/v2 v2.4.0
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20201113171705-d219536bb9fd
+	k8s.io/kube-openapi => github.com/liggitt/kube-openapi v0.0.0-20210113222446-36865573704e
```
Comment (dims) [inline go.mod:513]: what's the story here @liggitt ? (temporary redirect to personal repo)?

Comment (liggitt) [reply go.mod:513]: just testing until https://github.com/kubernetes/kube-openapi/pull/222 merges, then will replace the WIP commit with a real one

Comment (liggitt) [reply go.mod]: done

**Comment by liggitt** [comment]
false alarm, had a bogus BUILD file import referencing an existing import by a different path

**Pull request closed without merging**